server/database/sql/create_triggers.sql

```diff
@@ -9,3 +9,33 @@ before
 update on tracker
 for each row
 execute procedure trigger_set_timestamp();
+
+CREATE TRIGGER insert_words_from_overview AFTER INSERT 
```

Comment:
@ChrisShep98 this is where it gets crazy. these triggers all work together to populate the tables our dashboard functions query from when the user submits their data

this one gets triggered **after** data is **inserted** into the tracker table. when it's triggered the insert words function gets run